src/main/java/ch/uzh/ifi/hase/soprafs23/websockets/WebSocketConfig.java

```diff
@@ -14,10 +14,12 @@
 @EnableWebSocket
 public class WebSocketConfig implements WebSocketConfigurer {
 
-    private final static String CHAT_ENDPOINT = "/chat";
-    private final static String CARD_ENDPOINT = "/cards";
-    private final static String TEAM_ENDPOINT = "/teams";
-    private final static String PAGE_ENDPOINT = "/pages";
+    private final static String CHAT_ENDPOINT = "/chat/{accessCode}";
+    private final static String CARD_ENDPOINT = "/cards/{accessCode}";
+    private final static String TEAM_ENDPOINT = "/teams/{accessCode}";
+    private final static String PAGE_ENDPOINT = "/pages/{accessCode}";
+    private final static String TIMER_ENDPOINT = "/timer/{accessCode}";
+    private final static String PREGAME_ENDPOINT = "/pregame/{accessCode}";
 
```

Comment:
Could you please change chat, timer to chats, timers. And I'd change pregame to pregametimers.